docs/manual/getting_started.rst

```diff
@@ -714,35 +818,34 @@ with a name made of the\ ``<ModuleName>_<DataTypeName>`` .
     } HelloWorldData_Msg;
 
 **Note:** When translated into a different programming language, the
-data has a different representation that is specific to the target
-language. For instance, as shown in chapter 7, in C++ the Helloworld
-data type is represented by a C++ class. This is the advantage of using
-a neutral language like IDL to describe the data model.It can be
+data has another representation specific to the target
+language. For instance, as shown in chapter 7, in C++, the Helloworld
+data type is represented by a C++ class. This advantage of using
+a neutral language like IDL to describe the data model. It can be
 translated into different languages that can be shared between different
-applications written in different programming languages.
+applications written in other programming languages.
 
 Generated files with the IDL compiler
 '''''''''''''''''''''''''''''''''''''
 
-In Cyclone DDS the IDL compiler is a C program that processes .idl files.
+In Cyclone DDS, the IDL compiler is a C program that processes .idl files.
```

Comment:
Should we decide on a consistency as part of a style guide when referring to files? I have seen `IDL files`, `.idl files`, and `idl files`.